src/sim_explorer/models.py

```diff
@@ -0,0 +1,24 @@
+from enum import Enum
+
+from pydantic import BaseModel
+
+
+class Temporal(Enum):
+    UNDEFINED = 0
```

Comment:
we could change enum base class from `Enum` to `IntEnum`.  This gives some added support for e.g. integer comparison etc.
```suggestion
class Temporal(IntEnum):
```